x/feegrant/testutil/expected_keepers_mocks.go

```diff
@@ -5,7 +5,7 @@
 package testutil
 
 import (
-	"context"
+	context "context"
```

Comment:
weird that the linter does not complain about this. but this isn't needed.

Reply:
this is a mocked file, so probably linter ignore them